ios/MullvadVPN/RelayList.swift

```diff
@@ -8,27 +8,40 @@
 //
 
 import Foundation
+import Network
 
-struct RelayList: Decodable {
-    struct Country: Decodable {
+struct RelayList: Codable {
```

Comment:
FYI turned this into `Codable` because it has to be cached on disk, hence must be serialisable.